### ISSUE TITLE
Problem determining Higgs potential parameters even for negligilbe value of SMEFT coefficient `phil3`

### ISSUE THREAD

**Issue by peterstangl** [wilson-eft/wilson#49]
As pointed out by @Joro20 in https://github.com/flav-io/flavio/issues/121, the determination of of the SM parameters fails for a non-zero value of the WC `phil3`. This problem seems to be unrelated to the size of the WC since even for negligible values like `1e-50`, an error is generated. I have checked that this is the case for any flavour indices of `phil3`. The error can be reproduces by 
```python
from wilson import Wilson
wcphil3_13 = Wilson({'phil3_13': 1e-50}, scale=1e3, eft='SMEFT', basis='Warsaw')
wcphil3_13.match_run(scale=160, eft='SMEFT', basis='Warsaw')
```

**Comment by DavidMStraub** [comment]
Oops :hand_over_mouth:

**Comment by Joro20** [comment]
Hello, is there any solution to this issue ?

**Comment by DavidMStraub** [comment]
I don't have time to look into it currently.

**Comment by peterstangl** [comment]
This seems to be related to issue #40. The coefficients entering in `wilson.run.smeft.smpar.m2Lambda_to_vMh2` (see below) can get small imaginary parts. 
https://github.com/wilson-eft/wilson/blob/1cfb40777822cfb03d04bf37f39ece52082d8745/wilson/run/smeft/smpar.py#L38-L48

The issue is fixed by using only the real values, i.e.
```python
def m2Lambda_to_vMh2(m2, Lambda, C):
    """Function to numerically determine the  physical Higgs VEV and mass
    given the parameters of the Higgs potential."""
    try:
        v = (sqrt(2 * m2 / Lambda) + 3 * m2**(3 / 2) /
             (sqrt(2) * Lambda**(5 / 2)) * C['phi'].real)
    except ValueError:
        v = 0
    Mh2 = 2 * m2 * (1 - m2 / Lambda * (3 * C['phi'].real - 4 * Lambda * C['phiBox'].real +
                                        Lambda * C['phiD'].real))
    return {'v': v, 'Mh2': Mh2}
```

**Comment by DavidMStraub** [comment]
Oh, good point! :+1:

**Comment by peterstangl** [comment]
Fixed by PR #53.
